src/Core/Vault/Services/Implementations/CipherService.cs

```diff
@@ -849,13 +849,28 @@ public async Task RestoreAsync(Cipher cipher, Guid restoringUserId, bool orgAdmi
         await _pushService.PushSyncCipherUpdateAsync(cipher, null);
     }
 
-    public async Task RestoreManyAsync(IEnumerable<CipherDetails> ciphers, Guid restoringUserId)
+    public async Task<ICollection<CipherOrganizationDetails>> RestoreManyAsync(IEnumerable<Guid> cipherIds, Guid restoringUserId, Guid? organizationId = null, bool orgAdmin = false)
```

Comment:
Is there a possibility that the `cipherIds` would be null or empty? If yes, we might need some null handling and an early return.